src/main/java/com/github/instagram4j/instagram4j/responses/usertags/UserTagsFeedResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.github.instagram4j.instagram4j.responses.usertags;
+
+import java.util.List;
+
+import com.github.instagram4j.instagram4j.models.media.Media;
+import com.github.instagram4j.instagram4j.responses.IGPaginatedResponse;
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+
+import lombok.Data;
+
+@Data
+public class UserTagsFeedResponse extends IGResponse implements IGPaginatedResponse{
+
+	private List<Media> items;

```

Comment:
Change `Media` ->  `TimelineMedia`. I believe the UserTags only contain media from TimelineMedia (from the main feed timeline in instagram). I may be mistaken though.